openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamHierarchy.tsx

```diff
@@ -134,13 +134,14 @@ const TeamHierarchy: FC<TeamHierarchyProps> = ({
 
   const expandableConfig: ExpandableConfig<Team> = useMemo(
     () => ({
+      ...getTableExpandableConfig<Team>(),
       onExpand: (isOpen, record) => {
         if (isOpen && isEmpty(record.children)) {
           onTeamExpand(false, record.fullyQualifiedName, true);
         }
       },
     }),
-    [onTeamExpand]
+    [onTeamExpand, getTableExpandableConfig]
```

Comment:
`getTableExpandableConfig` is external function so no need to be added here

Reply:
My bad, I will remove that.